package.json

```diff
@@ -80,7 +80,7 @@
     "ts-loader": "^2.3.2",
     "tslint": "^5.5.0",
     "tslint-eslint-rules": "^4.0.0",
-    "typescript": "^2.2.1",
+    "typescript": "^2.4.2",
     "webpack": "^3.4.1",
     "webpack-dev-server": "^2.6.1"
   },
```

Comment:
Please update the new npm version of _"mendix-client"_

Reply:
Was done